image/cli-base/Dockerfile

```diff
@@ -13,29 +14,37 @@ RUN dnf update -y --skip-broken --nobest &&\
 
 # 2. Upgrade pip, install wheel, then install Python modules
 USER default
+
 COPY --chown=default:root install /tmp/install
 RUN python3 -m pip install --no-cache-dir --upgrade pip setuptools wheel &&\
-    python3 -m pip install --no-cache-dir -r /tmp/install/requirements.txt
+    pip3 install -r /tmp/install/requirements.txt --find-links /tmp/install
+    #python3 -m pip install --no-cache-dir -r /tmp/install/requirements.txt
 
 # 3. Install required tools & Ansible collections
 USER root
+
 RUN umask 0002 && \
-    bash /tmp/install/install-oc.sh && \
-    bash /tmp/install/install-ibmpak.sh && \
-    bash /tmp/install/install-ibmcloud.sh && \
-    bash /tmp/install/install-skopeo.sh && \
-    bash /tmp/install/install-aws.sh && \
-    bash /tmp/install/install-helm.sh && \
-    bash /tmp/install/install-mongo-tools.sh && \
-    bash /tmp/install/install-yq.sh && \
-    bash /tmp/install/install-argocd.sh && \
-    bash /tmp/install/install-rosa.sh && \
-    bash /tmp/install/install-rclone.sh && \
-    rm -rf /tmp/install && \
+    bash /tmp/install/install-oc.sh  --target-platform $ARCHITECTURE && \
+    bash /tmp/install/install-ibmpak.sh  --target-platform $ARCHITECTURE && \
+    bash /tmp/install/install-ibmcloud.sh  --target-platform $ARCHITECTURE && \
+    bash /tmp/install/install-skopeo.sh  --target-platform $ARCHITECTURE && \
+
+    bash /tmp/install/install-helm.sh  --target-platform $ARCHITECTURE && \
+    bash /tmp/install/install-mongo-tools.sh  --target-platform $ARCHITECTURE && \
+    bash /tmp/install/install-yq.sh  --target-platform $ARCHITECTURE && \
+    bash /tmp/install/install-argocd.sh --target-platform $ARCHITECTURE
+
+RUN if [ "${ARCHITECTURE}" == "amd64" ] ; then \
+         bash /tmp/install/install-aws.sh; \
+         bash /tmp/install/install-rosa.sh; \
+         bash /tmp/install/install-rclone.sh; \
```

Comment:
rclone is needed for the backup/restore. Is that not going to be available in s390x?

Reply:
Initially, yes.  

> Initially we are going to state limited support for running on s390x ... the README in this repo needs to be updated with the details of what restrictions are in place in the s390x image ... e.g. we're expecting that neither aws or rclone will be included on the s390x base image; and when we build the s390x cli image we will update it's README to document the features of the cli that are not supported on s390x.